tools/lib/provision.py

```diff
@@ -174,7 +174,7 @@
 BUILD_TSEARCH_FROM_SOURCE = False
 BUILD_PGROONGA_FROM_SOURCE = False
 if vendor in ["Ubuntu", "Debian"]:
-    if codename in ("cosmic", "disco"):
+    if codename in ("18.10", "19.04"):
```

Comment:
Although Ubuntu and Debian version numbers happen not to overlap, it would be less confusing to avoid relying on that: `if vendor == "ubuntu" and version in ("18.10", "19.04")`.

Reply:
6th commit.